src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -355,28 +347,23 @@ def killJob(self, jobIDList):
   def getCEStatus(self):
     """ Method to return information on running and pending jobs.
     """
+    if not self.useLocalSchedd:
+      return S_ERROR('Failed to get CE status: method not supported when using a remote schedd.')
+
+    cmd = ['condor_q', '-totals', '-json']
```

Comment:
this would have to be something like this: 
`condor_q --totals --json  -constraint 'regexp("numep-grid1.research.northwestern.edu", GridResource)'`
(replace by ce URL)

And use the entries `Running` and `Idle` instead my `MyRunning` and `MyIdle`


```
$ condor_q --totals --json  -constraint 'regexp("numep-grid1.research.northwestern.edu", GridResource)'
[
{
  "AllusersCompleted": 0,
  "AllusersHeld": 0,
  "AllusersIdle": 3166,
  "AllusersJobs": 3331,
  "AllusersRemoved": 159,
  "AllusersRunning": 6,
  "AllusersSchedulerCompleted": 0,
  "AllusersSchedulerHeld": 0,
  "AllusersSchedulerIdle": 0,
  "AllusersSchedulerJobs": 0,
  "AllusersSchedulerRemoved": 0,
  "AllusersSchedulerRunning": 0,
  "AllusersSuspended": 0,
  "Completed": 0,
  "ErrorCode": 0,
  "Held": 0,
  "Idle": 216,
  "Jobs": 216,
  "MyCompleted": 0,
  "MyHeld": 0,
  "MyIdle": 3166,
  "MyJobs": 3331,
  "MyName": "dirac",
  "MyRemoved": 159,
  "MyRunning": 6,
  "MySchedulerCompleted": 0,
  "MySchedulerHeld": 0,
  "MySchedulerIdle": 0,
  "MySchedulerJobs": 0,
  "MySchedulerRemoved": 0,
  "MySchedulerRunning": 0,
  "MySuspended": 0,
  "MyType": "Summary",
  "Removed": 0,
  "Running": 0,
  "SchedulerCompleted": 0,
  "SchedulerHeld": 0,
  "SchedulerIdle": 0,
  "SchedulerJobs": 0,
  "SchedulerRemoved": 0,
  "SchedulerRunning": 0,
  "ServerTime": 1631794040,
  "Suspended": 0
}
]
```


But maybe just replace the call to the DB by the call to the service (as @fstagni says)  in all cases if it only works for local schedd